python/metatensor-torch/metatensor/torch/documentation.py

```diff
@@ -608,20 +606,23 @@ def add_gradient(self, parameter: str, gradient: "TensorBlock"):
         Add gradient with respect to ``parameter`` in this block.
 
         :param parameter:
-            add gradients with respect to this ``parameter`` (e.g.
-            ``positions``, ``cell``, ...)
+            add gradients with respect to this ``parameter`` (e.g. ``positions``,
+            ``cell``, ...)
 
         :param gradient:
-            a :py:class:`TensorBlock` whose values contain the gradients with
-            respect to the ``parameter``. The labels of the gradient
-            :py:class:`TensorBlock` should be organized as follows: its
-            ``samples`` must contain ``"sample"`` as the first label, with
-            values containing the index of the corresponding ``samples`` in the
-            :py:class:`TensorBlock` containing values; its components must
-            contain at least the same components as the :py:class:`TensorBlock`
-            containing values, with any additional components coming before
-            those; its properties must match exactly those of the
-            :py:class:`TensorBlock` containing values.
+            a :py:class:`TensorBlock` whose values contain the gradients with respect to
```

Comment:
if you chose to take the suggested changes from the doc above, then here the changes would also need to be applied